pdr_backend/analytics/test/test_get_predictions_info.py

```diff
@@ -5,15 +5,14 @@
 from pdr_backend.lake.table import Table
 from pdr_backend.analytics.get_predictions_info import get_predictions_info_main
 from pdr_backend.ppss.ppss import mock_ppss
+from pdr_backend.lake.test.conftest import _clean_up_persistent_data_store
```

Comment:
it it's importable, it shouldn't start with an underscore. That is the convention for strictly module functions.